docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -208,6 +208,181 @@ The content provided in your app must be easily understandable to all users. Tha
+- If an app forces horizontal scrolling to read resized text, it would also fail.
+- Defining text sizes in pixels (px) rather than sp would prevent proper scaling and lead to accessibility issues.
+
+**Bad Example:** Fixed Height with Text Encapsulation and limiting the line count
+
+```
+<TextView
+    android:id="@+id/fixed_text"
+    android:layout_width="wrap_content"
+    android:layout_height="50dp" <!-- Fixed height -->
+    android:maxLines="2"  <!-- Limits the text to 2 lines -->
+    android:ellipsize="end" <!-- Adds "..." at the end if text is too long -->
+    android:text="This is a long piece of text that might be cut off when resized."
+    android:textSize="16sp" />
+```
+
+```kotlin
+Box(modifier = Modifier.height(50.dp)) {
+    Text(
+        text = "This is a long piece of text that might be cut off when resized.",
+        fontSize = 16.sp,
+        maxLines = 2,  // Limits the text to 2 lines
+        overflow = TextOverflow.Ellipsis  // Adds "..." at the end if text is too long
+    )
+}
+```
+
+**Solution:** 
+
+```
+<TextView
+    android:id="@+id/fixed_text"
+    android:layout_width="wrap_content"
+    android:minHeight="50dp"  <!-- Minimum height but flexible -->
+    android:text="This is a long piece of text that might be cut off when resized."
+    android:textSize="16sp" />
+```
+
+```kotlin
+Box(modifier = Modifier
+    .minHeight(50.dp)  // Minimum height but flexible
+    .wrapContentHeight()) {
+    Text(
+        text = "This is a long piece of text that will resize appropriately.",
+        fontSize = 16.sp
+    )
+}
+```
+
+**Important Point:** Handling Large Text Without Scrollable Container
+
+When presenting large text or content that may exceed the visible area of the screen, it is crucial to provide a way for users to read all the content without losing access. This is especially important for users who might need larger text for better readability.
+
+```
+<!-- Using ScrollView to contain large text -->
+<ScrollView
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/large_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="This is a very long piece of text that might not fit on the screen when enlarged. Users need to be able to scroll to read all of it."
+        android:textSize="24sp" />
+</ScrollView>
+```
+
+```kotlin
+// Compose example with scrollable text
+val largeText = "This is a very long piece of text that might not fit on the screen when enlarged. Users need to be able to scroll to read all of it."
+
+Column(modifier = Modifier.verticalScroll(rememberScrollState())) {
+    Text(
+        text = largeText,
+        fontSize = 24.sp  // Large font size
+    )
+}
+```
+
+---
+
+### Images of Text
+
+*This guideline covers point 1.4.5 Images of Text - Level AA of the WCAG standard.*
+
+:white_check_mark: **Success criteria**
```

Comment:
I think you should mention contentDescription here for image view and compose alternative. Check the iOS for more info